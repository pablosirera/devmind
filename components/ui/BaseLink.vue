<template>
  <a class="bg-blue-200 p-2 rounded" href="https://pablosirera.com"
    >Ir a la web</a
  >
</template>

<script>
export default {
  name: 'BaseLink',
}
</script>
