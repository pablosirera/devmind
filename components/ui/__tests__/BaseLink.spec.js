import { shallowMount } from '@vue/test-utils'
import BaseLink from '@/components/ui/BaseLink.vue'

describe('UI Components -> BaseLink', () => {
  const build = () => {
    const wrapper = shallowMount(BaseLink)

    return {
      wrapper,
    }
  }

  test('should be render', () => {
    const { wrapper } = build()
    expect(wrapper.vm).toBeTruthy()
  })

  test('should match snapshot', () => {
    const { wrapper } = build()
    expect(wrapper.element).toMatchSnapshot()
  })
})
