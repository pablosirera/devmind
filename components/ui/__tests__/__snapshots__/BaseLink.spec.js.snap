// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI Components -> BaseLink should match snapshot 1`] = `
<a
  class="bg-blue-200 p-2 rounded"
  href="https://pablosirera.com"
>
  
  Ir a la web

</a>
`;
